packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -787,6 +788,32 @@ class MediaQuery extends InheritedWidget {
     );
   }
 
+  /// Creates a [_MediaQueryFromWindow] which builds and updates a
```

Comment:
```suggestion
  /// Creates a [MediaQuery] from the [Window] which builds and updates a
```

Here and elsewhere. Docs for private classes are not generated, so in the api docs this would just make a broken link.

Reply:
Makes sense, removed the public references to the private class.